CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Please note that compatibility for 0.x releases (software or repositories) isn't guaranteed. Kart is evolving quickly and things will change. However, we aim to provide the means to upgrade existing repositories between 0.x versions and to 1.0.
 
 _When adding new entries to the changelog, please include issue/PR numbers wherever possible._
+## 0.11.2 (UNRELEASED)
```

Comment:
You'll need to merge or rebase this with your other change now which I've just merged to master for you